tests/e2e/utils/pages/price-benchmark.js

```diff
@@ -70,4 +70,76 @@ export default class PriceBenchmarkPage extends MockRequests {
 			.locator( '../..' )
 			.locator( '.woocommerce-summary__item-value span' );
 	}
+
+	/**
+	 * Retrieves the locator for the "Change Price" modal element on the page.
+	 *
+	 * @return {import('@playwright/test').Locator} The "Change Price" modal element.
+	 */
+	getChangePriceModal() {
+		return this.page.locator( '.gla-change-price-modal' );
+	}
+
+	/**
+	 * Retrieves the price input field within the "Change Price" modal.
+	 *
+	 * @return {import('@playwright/test').Locator} The element handle for the "New Price" input field.
+	 */
+	getPriceInputModal() {
+		return this.getChangePriceModal().getByLabel( 'New Price' );
+	}
+
+	/**
+	 * Retrieves the "Change Price" button element within the change price modal.
+	 *
+	 * @return {import('@playwright/test').Locator} The "Change Price" button element within the modal.
+	 */
+	getChangePriceButtonModal() {
```

Comment:
Nit: This seems to better align with other names
```suggestion
	getChangePriceModalButton() {
```

Reply:
👍